tests/IndexBuilderTest.php

```diff
@@ -169,4 +171,56 @@ public function testPurgeAndCloseOldIndices(): void
             $this->assertStringContainsStringIgnoringCase('closed', $e->getMessage());
         }
     }
+
+    public function testSlowDownRefresh(): void
+    {
+        $indexBuilder = $this->getIndexBuilder(__DIR__.'/configs_analysis');
+        $index = $indexBuilder->createIndex('hop');
+        $indexBuilder->slowDownRefresh($index);
+        $this->assertEquals('60s', $index->getSettings()->getRefreshInterval());
+    }
+
+    public function testSpeedUpRefresh(): void
+    {
+        $indexBuilder = $this->getIndexBuilder(__DIR__.'/configs_analysis');
+        $index = $indexBuilder->createIndex('hop');
+        $indexBuilder->speedUpRefresh($index);
+        $this->assertEquals('1s', $index->getSettings()->getRefreshInterval());
+    }
+
+    public function testMigrate(): void
+    {
+        $indexName = mb_strtolower(__FUNCTION__);
+
+        $dto = new MigrateDTO();
+        $dto->bar = 'I like unicorns.';
+        $dto->foo = 'Why is the sky blue?';
+
+        $indexer = $this->getClient()->getIndexer();
+
+        $indexer->scheduleIndex($indexName, new Document('f', $dto));
+        $indexer->flush();
+
+        $indexer->refresh($indexName);
+
+        $client = $this->getClient();
+        $document = $client->getIndex($indexName)->getDocument('f');
+        $this->assertInstanceOf(Document::class, $document);
+
+        $indexBuilder = $this->getIndexBuilder(__DIR__.'/configs');
+        $newIndex = $indexBuilder->createIndex('empty');
+
+        $indexBuilder->migrate($client->getIndex($indexName), $newIndex);
+        usleep(1200000); // 1,2 second
```

Comment:
This is kind of random, but we can keep this while waiting for the "pooling task API" feature, cf my other comment.